db/schema.rb

```diff
@@ -437,7 +437,6 @@
     t.datetime "updated_at", precision: 6, null: false
     t.integer "output_payment_percantage", default: 60
     t.integer "service_fee_percentage", default: 40
-    t.integer "service_fee_contribution_percentage", default: 40
```

Comment:
This was a fresh schema dump. Does this look wrong to anyone?

Reply:
@blackrat was that something you had on a local branch?

Anyway, I don't see it in any migrations, or used for anything, so I'm fine cutting it from schema.